athena/utils.py

```diff
@@ -2,6 +2,7 @@
 """
 import numpy as np
 from scipy.optimize import linprog
+from ezyrb import GPR
```

Comment:
I am not sure about this. If we need just GPR, is it easier for the user to install directly GPy